src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { memo } from 'react';
+import classNames from 'classnames';
+import { FilterBy } from '../../utils/enums';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  statusFilter: FilterBy,
+  onFilterChange: (filter: FilterBy) => void,
+  onDeleteTodo: (todoId: number) => void
+};
+
+export const TodoFilter: React.FC<Props> = memo(
+  ({
+    todos,
+    statusFilter,
+    onFilterChange,
+    onDeleteTodo,
+  }) => {
+    const activeTodosNumber = todos.filter(todo => !todo.completed).length;
+    const hasCompletedTodos = todos.length !== activeTodosNumber;
+
+    const clearCompleted = () => {
```

Comment:
`clearCompleted` function can be taken outside the component and used as a helper.